documentation/examples/21-miscellaneous/02-immutable-data/MutableTodo.svelte

```diff
@@ -13,7 +13,8 @@
 
 <!-- the text will flash red whenever
 		the `todo` object changes -->
-<div bind:this={div} on:click>
+<!-- svelte-ignore a11y-click-events-have-key-events -->
+<div bind:this={div} on:click role="button" tabindex='0'>
```

Comment:
As above, we should add a button to complete todos instead of ignoring the warning